scio-coders-macros/src/main/scala/com/spotify/scio/coders/CoderMacros.scala

```diff
@@ -21,15 +21,14 @@ import scala.reflect.macros._
 
 private[coders] object CoderMacros {
 
-  var verbose = true
-  val reported: scala.collection.mutable.Set[(String, String)] =
+  private[this] var verbose = true
+  private[this] val Reported: scala.collection.mutable.Set[(String, String)] =
```

Comment:
Nitpicking: Why did you uppercase this ?

Reply:
Ah! nice catch i'll revert this since it's mutable